diff -Nurp hping2-rc3/waitpacket.c hping2-rc3-fix/waitpacket.c
--- hping2-rc3/waitpacket.c	2002-12-13 17:01:57.000000000 +0200
+++ hping2-rc3-fix/waitpacket.c	2005-08-14 14:47:28.000000000 +0300
@@ -499,7 +499,7 @@ found:
 	printf("  TCP timestamp: tcpts=%u\n", tstamp);
 	if (last_tstamp && !opt_waitinusec) {
 		int tsdiff = (tstamp - last_tstamp) / sending_wait;
-		int hz_set[] = { 2, 10, 100, 1000, 0 };
+		int hz_set[] = { 2, 10, 100, 250, 1000, 0 };
 		int hzdiff = -1;
 		int hz = 0, sec;
 		int days, hours, minutes;
